massa-client/src/cmds.rs

```diff
@@ -151,6 +154,17 @@ pub enum Command {
         message = "send coins from a wallet address"
     )]
     send_transaction,
+
+    #[strum(
+        ascii_case_insensitive,
+        props(
+            args = "SenderAddress PathToBytecode MaxGas GasPrice Coins Fee",
+            todo = "[unstable] "
+        ),
+        message = "create and sends an operation containing byte code"
```

Comment:
Missing a "s" and an explanation

Creates and sends an operation containing bytecode to execute on-chain

Reply:
For consistency 'create and send'